mini_pupper_control/mini_pupper_control/StanfordQuadruped/pupper/Config.py

```diff
@@ -0,0 +1,251 @@
+# MIT License
```

Comment:
It is defined in mini_pupper_bsp or mini_pupper_2_bsp, so I think this file is not necessary.

Reply:
Since StanfordQuadruped is deeply coupled with mini_pupper_bsp, it will be a significant change to remove this Config.py file. I prefer to reserve this file and make it a PLAN to remove this in the future.

Reply:
I agree that this software depends on mini_pupper_bsp.
And I believe we can refer to Config.py in mini_pupper_bsp.
Could you please explain in more detail why it cannot be removed?